### PR TITLE
feat(drone-nomad): Add drone nomad plugin

### PR DESCRIPTION
**Pull request by marcotuna**
Add documentation for drone nomad plugin.
This plugins allows deploying services to nomad via a template file, this could be integrated on the pipeline to deploy services at the end of the build process as this allows to pass variables on the nomad template for drone to replace them.
The logo is from the Hashicorp official website for Nomad, it was not modified and is according to the trademark.
https://www.hashicorp.com/brand

### CODE REVIEW

File: content/loq9/drone-nomad/index.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+---
+date: 2019-10-22T00:00:00+00:00
+title: Nomad
+author: loq9
+tags: [ nomad, orchestration ]
+logo: nomad.svg
+repo: loq9/drone-nomad
+image: plugins/nomad
```
Comment (tboerger) [inline content/loq9/drone-nomad/index.md]: This image name can't be true, only images built at the drone-plugins org are published with that name.

Comment (tboerger) [reply content/loq9/drone-nomad/index.md]: `loq9/drone-nomad` should be right.

Comment (marcotuna) [reply content/loq9/drone-nomad/index.md]: You're right, I followed the drone official plugins to do this one. I will change it

**Comment by marcotuna** [comment]
@tboerger, done. I've changed the image name from "plugins/nomad" to "loq9/drone-nomad"

**Pull request merged**